modal_proto/api.proto

```diff
@@ -2879,6 +2879,71 @@ message VolumeMount {
   bool read_only = 4;
 }
 
+message VolumePutFiles2Request {
+  // List of files, sorted lexicographically by `path`.
```

Comment:
Curious as to why the list needs to be sorted here?

Reply:
I think it is good practice to always have a well-defined structure of requests/responses, see earlier discussion here: https://github.com/modal-labs/modal-client/pull/3004#discussion_r2032750234